src/plots/cartesian/graph_interact.js

```diff
@@ -17,8 +17,9 @@ var Plotly = require('../../plotly');
 var Lib = require('../../lib');
 var Events = require('../../lib/events');
 
-var prepSelect = require('./select');
 var constants = require('./constants');
+var dragBox = require('./dragbox');
+var dragElement = require('../../components/dragElement');
```

Comment:
@alexcjohnson here's why the tests are failing on CircleCI,

The file name is `dragelement.js`. On OSX, `dragElement` must get resolved as `dragelement`, but not on Ubuntu.